packages/react/src/components/buttons/button.tsx

```diff
@@ -78,7 +121,9 @@ export const Button = forwardRef<HTMLButtonElement, PropsWithChildren<ButtonProp
             {...props /* eslint-disable-line react/jsx-props-no-spreading *//* To spread aria-* and data-* */}
         >
             {children}
+            {leftIconName && renderIcon(leftIconName)}
```

Comment:
J'irais plus avec une syntaxe comme ça plutôt que de passer un booléen à une fonction. Ça fait un peu moins d'indirection et de if-else

```suggestion
            {leftIconName && (
                <LeftIcon
                    aria-hidden="true"
                    data-testid="left-icon"
                    name={leftIconName}
                    size={iconSize}
                />
            )}
```

Reply:
Sure. J'étais pas certain qu'on me dise que c'est plus efficace de mettre ça dans une fonction si j'implémentais ça simple de même. 🤷‍♂️